src/SASjs.ts

```diff
@@ -914,10 +912,12 @@ export default class SASjs {
       this.sasjsConfig.serverUrl = this.sasjsConfig.serverUrl.slice(0, -1)
     }
 
-    this.requestClient = new RequestClient(
-      this.sasjsConfig.serverUrl,
-      this.sasjsConfig.allowInsecureRequests
-    )
+    if (!this.requestClient) {
+      this.requestClient = new RequestClient(
```

Comment:
If the request client has already been instantiated, we should update the serverUrl and allowInsecureRequests parameters in the existing instance.